lomap/graphgen.py

```diff
@@ -106,7 +111,8 @@ def __init__(self, dbase):
         self.max_nodes = 100
 
         # The maximum threshold distance in A unit used to select if a molecule is depicted
```

Comment:
yes, for some big molecules with the small max_mol_size, it will not be depicted.